File: Habitica/src/main/java/com/habitrpg/android/habitica/models/tasks/TaskGroupPlan.java
package com.habitrpg.android.habitica.models.tasks;


import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;

public class TaskGroupPlan extends RealmObject {

    @PrimaryKey
    String task_id;

    public boolean approvalRequested, approvalApproved, approvalRequired;
}

File: shared/src/androidMain/kotlin/com/habitrpg/shared/habitica/models/tasks/TaskGroupPlan.kt
package com.habitrpg.shared.habitica.models.tasks


import io.realm.RealmObject
import io.realm.annotations.PrimaryKey

actual open class TaskGroupPlan : RealmObject() {

    @PrimaryKey
    internal actual var taskID: String? = null

    actual var approvalRequested: Boolean = false
    actual var approvalApproved: Boolean = false
    actual var approvalRequired: Boolean = false
}
